docs/tutorials/tabular_prediction/tabular-multimodal-text-others.md

```diff
@@ -7,7 +7,7 @@ This type of data, i.e., data which contains text and other features, is prevale
 For example, when building a sentiment analysis model of users' tweets, we can not only use the raw text in the 
 tweets but also other features such as the topic of the tweet and the user profile. In the following, 
 we will investigate different ways to ensemble the state-of-the-art (pretrained) language models in AutoGluon TextPrediction 
-with all of the other models used in AutoGluon TabularPrediction. 
+with all the other models used in AutoGluon TabularPrediction. 
```

Comment:
"AutoGluon TabularPrediction" 
->
"AutoGluon's TabularPredictor"

since the former no longer exists

Reply:
Added